plugin/rpcplugin/sandbox/sandbox_linux.go

```diff
@@ -267,7 +267,7 @@ func pivotRoot(newRoot string) error {
 func dropInheritableCapabilities() error {
 	type capHeader struct {
 		version uint32
-		pid     int
+		pid     int32
```

Comment:
Added to master here: https://github.com/mattermost/mattermost-server/pull/8389. Should also go into the release branch.